mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -76,28 +98,49 @@ CLASS aii net/minecraft/entity/LivingEntity
 	METHOD b heal (F)V
 		ARG 1 amount
 	METHOD b getStackInHand (Lagt;)Lbbq;
-	METHOD b getPotionEffect (Lahr;)Laht;
+	METHOD b applyArmorToDamage (Lahi;F)F
+		ARG 1 source
+		ARG 2 damage
+	METHOD b getStatusEffect (Lahr;)Laht;
 	METHOD b getEquippedStack (Laie;)Lbbq;
 	METHOD b getActiveEyeHeight (Lais;Laia;)F
 	METHOD b onEquipStack (Lbbq;)V
 		ARG 1 stack
+	METHOD b spawnEatingParticlesAndPlayEatingSoundOrPlayDrinkingSound (Lbbq;I)V
```

Comment:
This is quite long.. 😄 
Could do something along the lines of `playConsumptionAnimation`